ts/src/pro/blockchaincom.ts

```diff
@@ -3,7 +3,7 @@
 import blockchaincomRest from '../blockchaincom.js';
 import { NotSupported, AuthenticationError, ExchangeError } from '../base/errors.js';
 import { ArrayCache, ArrayCacheBySymbolById, ArrayCacheByTimestamp } from '../base/ws/Cache.js';
-import type { IndexType, Int, Str, OrderBook, Order, Trade, Ticker, OHLCV, Balances } from '../base/types.js';
```

Comment:
is this blockchaincom update on purpose?

Reply:
Yes, with the change of parseOrderBook, I was able to clean up and delete `parseCountedOrderBook` from blockchaincom.  And after that IndexType is not used.